src/core_plugins/kibana/public/management/sections/indices/source_filters/source_filters.js

```diff
@@ -0,0 +1,119 @@
+import { find, each, escape, invoke, size, without } from 'lodash';
+
+import uiModules from 'ui/modules';
+import Notifier from 'ui/notify/notifier';
+import FieldWildcardProvider from 'ui/field_wildcard';
+
+import controlsHtml from 'plugins/kibana/management/sections/indices/source_filters/controls.html';
+import filterHtml from 'plugins/kibana/management/sections/indices/source_filters/filter.html';
+import template from './source_filters.html';
+import './source_filters.less';
+
+const notify = new Notifier();
+
+uiModules.get('kibana')
+.directive('sourceFilters', function (Private, $filter) {
+  const angularFilter = $filter('filter');
+  const { fieldWildcardMatcher } = Private(FieldWildcardProvider);
+  const rowScopes = []; // track row scopes, so they can be destroyed as needed
+
+  return {
+    restrict: 'E',
+    scope: {
+      indexPattern: '='
+    },
+    template,
+    controllerAs: 'sourceFilters',
+    controller: class FieldFiltersController {
+      constructor($scope) {
+        if (!$scope.indexPattern) {
+          throw new Error('index pattern is required');
+        }
+
+        $scope.perPage = 25;
+        $scope.columns = [
+          {
+            title: 'filter'
+          },
+          {
+            title: 'matches',
+            sortable: false,
+            info: 'The known fields that match the filter'
+          },
+          {
+            title: 'controls',
+            sortable: false
+          }
+        ];
+
+        this.$scope = $scope;
+        this.saving = false;
+        this.editing = null;
+        this.newValue = null;
+        this.placeHolder = 'source filter, accepts wildcards (e.g., `user*` to filter fields starting with \'user\')';
+
+        $scope.$watchMulti([ '[]indexPattern.sourceFilters', '$parent.fieldFilter' ], () => {
+          invoke(rowScopes, '$destroy');
```

Comment:
hmmm, in that case let's just leave it in. Might as well be consistent between sections